src/librbd/cache/PassthroughImageCache.cc

```diff
@@ -57,11 +57,11 @@ void PassthroughImageCache<I>::aio_discard(uint64_t offset, uint64_t length,
 }
 
 template <typename I>
-void PassthroughImageCache<I>::aio_flush(Context *on_finish) {
+void PassthroughImageCache<I>::aio_flush(librbd::io::FlushSource flush_source, Context *on_finish) {
   CephContext *cct = m_image_ctx.cct;
   ldout(cct, 20) << "on_finish=" << on_finish << dendl;
 
-  m_image_writeback.aio_flush(on_finish);
+  m_image_writeback.aio_flush(io::FLUSH_SOURCE_WRITEBACK, on_finish);
```

Comment:
Nit: why is this hard-coded to `WRITEBACK`? Technically, this is the API normally invoked for user-generated flush requests and the `flush` method below was for librbd-internal requests.

Reply:
+1

Reply:
Done